compiler/rustc_feature/src/accepted.rs

```diff
@@ -93,6 +93,8 @@ declare_features! (
     (accepted, c_unwind, "1.81.0", Some(74990)),
     /// Allows `#[cfg_attr(predicate, multiple, attributes, here)]`.
     (accepted, cfg_attr_multi, "1.33.0", Some(54881)),
+    /// Allows the use of `#[cfg(<true/false>)]`.
+    (accepted, cfg_boolean_literals, "CURRENT_RUSTC_VERSION", Some(131204)),
```

Comment:
Question: how does this interact with the cli interface, i.e. what if I'm a sneaky user and wrote

```rs
// FILENAME: foo.rs
// CLI:      rustc --cfg=true foo.rs

#[cfg(true)]
fn main() {}
```

Is this permitted? If so, what's the behavior? It would be good to add a test case for this.

For instance, on nightly, `rustc --cfg=true` on

```rs
#![feature(cfg_boolean_literals)]
#[cfg(true)]
fn main() {}
```

Successfully compiles. (It produces a warning that `main` is unused, but that's because `dead_code` vs cfg-gated code.)

Reply:
> Is this permitted? If so, what's the behavior? It would be good to add a test case for this.

I believe we have a test for it, and what it does is that `true` (in the CLI arg) is treated as an identifier and to access it you need to use the raw syntax, that is `r#true`.

Reply:
I'm having a bit of trouble finding this specific test, so it might be good linking it in the stabilization report itself :D